src/content/1.7/development/Multistore/configuration-forms/_index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Configuration forms
+menuTitle: Configuration forms
+summary: "Configuration forms when multistore is enabled"
+weight: 50
+---
+
+# Configuration forms in multistore context
+{{< minver v="1.7.8" title="true" >}}
+
+Since Prestashop 1.7.8, when multistore is enabled, migrated configuration forms can display a checkbox before the fields in order to specify if the configuration value is set for the current context shop/group, or if it is a value inherited from all contexts, or from a shop group.
```

Comment:
```suggestion
Since Prestashop 1.7.8, when multistore is enabled, migrated configuration forms can display a checkbox before fields in order to specify if the configuration value is set for the current context shop/group, or if it is a value inherited from all contexts, or from a shop group.
```